doc/fbchkdoc/delextra.bas

```diff
@@ -79,77 +140,77 @@ sub DeleteExtraFiles _
 				end if
 			end if
 		end if
+
 		d = dir()
 	wend
 
-end sub
+	if( bHTML ) then
+		loghtml( "" )
+		loghtml( "</body></html>" )
+	end if
 
+end sub
 
 '' --------------------------------------------------------
 '' MAIN
 '' --------------------------------------------------------
 
-dim as string cache_dir, def_cache_dir, web_cache_dir, dev_cache_dir
-dim as integer i = 1, isgit = FALSE, nodelete = FALSE
+'' from cmd_opts.bas
+extern cmd_opt_help as boolean
```

Comment:
`extern` declarations can be moved into a header file, like function declarations, to avoid mismatching signatures.